pages/blueprints/index.js

```diff
@@ -40,9 +40,21 @@ const messages = defineMessages({
   emptyTitle: {
     defaultMessage: "No Blueprints"
   },
-  errorTitle: {
+  errorGenericTitle: {
     defaultMessage: "An Error Occurred"
   },
+  errorInactiveTitle: {
+    defaultMessage: "Image Building Service is Not Active"
+  },
+  errorInactivePrimary: {
+    defaultMessage: "Start"
+  },
+  errorInactiveSecondary: {
+    defaultMessage: "Troubleshoot"
+  },
+  errorInactiveCheckbox: {
```

Comment:
These are really good points. I used "error" because this is the result of an error returned by the api. I used "inactive" because the error is returned when the service is not "active". I see your point about the variable name not reflecting the checkbox purpose. My thinking here is that all string variables that are used in the EmptyState component when this error is returned should be prefaced with the name of that state (e.g. "errorInactive") so that it's easy to see they're all related, and the rest of the string is just the UI element in that state (e.g. "Title", "Primary", "Checkbox") so that it's easy to visualize how these strings are being rendered.